docs/collector.mssql.md

```diff
@@ -24,80 +24,80 @@ Name | Description | Type | Labels
-`wmi_mssql_bufman_extension_page_writes` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_free_list_stalls` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_integral_controller_slope` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_lazywrites` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_page_life_expectancy_seconds` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_page_lookups` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_page_reads` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_page_writes` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_read_ahead_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_read_ahead_issuing_seconds` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_target_pages` | _Not yet documented_ | counter | `instance`
+`wmi_mssql_bufman_extension_outstanding_io` | Percentage of the buffer pool extension paging file occupied by buffer manager pages | counter | `instance`
+`wmi_mssql_bufman_extension_page_evictions` | Number of pages evicted from the buffer pool extension file per second | counter | `instance`
+`wmi_mssql_bufman_extension_page_reads` | Number of pages read from the buffer pool extension file per second | counter | `instance`
+`wmi_mssql_bufman_extension_page_unreferenced_seconds` | Average seconds a page will stay in the buffer pool extension without references to it | counter | `instance`
+`wmi_mssql_bufman_extension_page_writes` | Number of pages written to the buffer pool extension file per second | counter | `instance`
+`wmi_mssql_bufman_free_list_stalls` | Indicates the number of requests per second that had to wait for a free page | counter | `instance`
+`wmi_mssql_bufman_integral_controller_slope` | The slope that integral controller for the buffer pool last used, times -10 billion | counter | `instance`
+`wmi_mssql_bufman_lazywrites` | Indicates the number of buffers written per second by the buffer manager's lazy writer | counter | `instance`
+`wmi_mssql_bufman_page_life_expectancy_seconds` | Indicates the number of seconds a page will stay in the buffer pool without references | counter | `instance`
+`wmi_mssql_bufman_page_lookups` | Indicates the number of requests per second to find a page in the buffer pool | counter | `instance`
+`wmi_mssql_bufman_page_reads` | Indicates the number of physical database page reads that are issued per second | counter | `instance`
+`wmi_mssql_bufman_page_writes` | Indicates the number of physical database page writes that are issued per second | counter | `instance`
+`wmi_mssql_bufman_read_ahead_pages` | Indicates the number of pages read per second in anticipation of use | counter | `instance`
+`wmi_mssql_bufman_read_ahead_issuing_seconds` | Time (microseconds) spent issuing readahead | counter | `instance`

```

Comment:
Is this seconds, per the name, or microseconds?

Reply:
I believe it should be seconds